meadow-client/src/Semantics.purs

```diff
@@ -423,8 +228,8 @@ valueParser = do
       idCommit <- bigIntParser
       pure (Committed idCommit)
     "Constant" -> do
-      bigInt <- bigIntParser
-      pure (Constant bigInt)
+      bigInteger <- bigIntParser
+      pure (Constant bigInteger)
```

Comment:
Worth changing these simple `do`-blocks to `Constant <$> bigInteger`?

Reply:
this is all going to be removed once we have some tests that can verify the new parser works with the semantics, I have left the old Parser here until then